src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -33,6 +37,11 @@ const BottomSheetBackdropComponent = ({
   onPress,
   style,
   children,
+  accessible: _providedAccessible = DEFAULT_ACCESSIBLE,
+  accessibilityRole: _providedAccessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
+  accessibilityLabel: _providedAccessibilityLabel = DEFAULT_ACCESSIBILITY_LABEL,
+  accessibilityHint: _providedAccessiblityHint = DEFAULT_ACCESSIBILITY_HINT,
+  ...rest
```

Comment:
Why is the spread of the `rest` props needed?

Reply:
It currently contains the animatedPosition-prop. I could just write it out instead of using the spread operator.